services/gitlab.service

```diff
@@ -18,7 +18,8 @@ ExecStartPre=/usr/bin/env bash -c "/usr/bin/docker run -d \
     --net protonet \
     --volume /data/gitlab/data:/home/git/data \
     -p \"$(/opt/bin/gitlab-network show):2222:22\" \
-    --env \"GITLAB_HOST=$(/opt/bin/gitlab-network show)\" \
+    --env \"GITLAB_HOST=gitlab.$(</etc/protonet/hostname).protonet.info\" \
+    --env \"GITLAB_SSH_HOST=$(/opt/bin/gitlab-network show)\" \
```

Comment:
awesome! didn't know we can configure both individually :)

however: we will have to come up with a way to set this to the custom domain that a user has set.
maybe this is a low hanging fruit for future sprints? @colszowka